vg/Makefile

```diff
@@ -4,7 +4,7 @@ runtime_fullpath = $(realpath runtime)
 build_tool = runtime-container.DONE
 git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../vg | cut -f1 -d " ")
 name = quay.io/ucsc_cgl/vg
-tag = 1.4.0--${git_commit}
+tag = 1.4.0--cb68327d76dd718dd155a6462866e65b093d4644
```

Comment:
This is supposed to be the commit of *this* repository, not a commit in the upstream VG repo. You are free to refer to an upstream commit in the tag but then this line would have to read 

```
tag = 1.4.0-cb68327--${git_commit}
```

Note the difference between single dash and double dash as a separator. Also note the use of the seven-digit abbreviated hash.